readme.md

```diff
@@ -128,9 +128,8 @@
   - [Discord API](https://discord.gg/api)
   - [Discord Feedback](https://discord.gg/discord-feedback)
   - [Discord Events](https://discord.gg/events)
-- URLS
+- Links
   - [Developer Documentation](https://discordapp.com/developers)
-  - [Discord Nitro](https://discordapp.com/nitro)
```

Comment:
Discord Nitro is important and relevant, it should stay here.

Reply:
Already fixed in my latest commit.